techniques/aria/ARIA17.html

```diff
@@ -1,84 +1,110 @@
+    &lt;span id="a2r2"&gt;No&lt;/span&gt;
+  &lt;/div&gt;
+&lt;/div&gt;
+&lt;input type="submit" value="Continue" id="continue_btn" name="continue_btn" /&gt;</pre>
+
+   <p class="working-example">Working example: <a href="../../working-examples/aria-grouping-related-fields/">using grouping roles to identify related form controls</a>.</p>
          
-      </section>
-   </section><section id="tests"><h2>Tests</h2>
-      <section class="procedure"><h3>Procedure</h3>
-         <p>For groups of related controls where the individual labels for each control do not provide a sufficient description, and an additional group level description is needed:
-</p>
-         <ol>
-            <li> Check that the group of logically related input or select elements are contained within an element with role=group. 
-</li>
-            <li> Check that this group has an accessible name defined using <code class="att">aria-label</code> or <code class="att">aria-labelledby</code>.
-</li>
-         </ol>
-      </section>
-      <section class="results"><h3>Expected Results</h3>
-         <ul>
-            <li>#1 and #2 are true.</li>
-         </ul>
-      </section>
-   </section><section id="related"><h2>Related Techniques</h2><ul>
+   </section>
+</section>
+<section id="tests">
+   <h2>Tests</h2>
+   <section class="procedure">
+      <h3>Procedure</h3>
+      <p>For groups of related controls where the individual labels for each control do not provide a sufficient description, and an additional group level description is needed:</p>
+      <ol>
+         <li>Check that the group of logically related input or select elements are contained within an element with <code class="prop">role=group</code>.</li>
+         <li>Check that this group has an accessible name defined using <code class="prop">aria-label</code> or <code class="prop">aria-labelledby</code>.</li>
+      </ol>
+   </section>
+   <section class="results">
+      <h3>Expected Results</h3>
+      <ul>
+         <li>#1 and #2 are true.</li>
+      </ul>
+   </section>
+</section>
+<section id="related">
+   <h2>Related Techniques</h2>
+   <ul>
       <li><a href="../html/H71">H71</a></li>
-   </ul></section><section id="resources"><h2>Resources</h2>
-      
-         <ul>
-            <li>
-                  <a href="https://www.w3.org/TR/wai-aria/#group">Accessible Rich Internet Applications (WAI-ARIA), The Roles Model</a>
-               </li>
-         </ul>
-      
-   </section></body></html>
\ No newline at end of file
+   </ul>
+</section>
+<section id="resources">
+   <h2>Resources</h2>
+   <ul>
+      <li>
+         <a href="https://www.w3.org/TR/wai-aria/#group">Accessible Rich Internet Applications (WAI-ARIA), The <code class="prop">group</code> role</a>
+      </li>
+      <li>
+         <a href="https://www.w3.org/TR/wai-aria/#group">Accessible Rich Internet Applications (WAI-ARIA), The <code class="prop">radiogroup</code> role</a>
```

Comment:
There is the separate link for radiogroup, https://www.w3.org/TR/wai-aria-1.1/#radiogroup

Currently ARIA 1.2 is in CR. I am not sure whether we can add ARIA 1.2 Radiogroup link instead. 

ARIA 1.2 Radio group has two more supported state and properties of 1)aria-errormessaeg and 2)aria-invalid. It also has 4 less inhertied states and properties.

Reply:
added the `radiogroup` link.